app/views/waste_exemptions_engine/shared/_manual_address.html.erb

```diff
@@ -46,14 +34,10 @@
   </fieldset>
 </div>
 
-<% if form.errors[:city].any? %>
-<div class="form-group form-group-error">
-<% else %>
-<div class="form-group">
-<% end %>
+<div class="form-group <%= "form-group-error" if form.errors[:ciry].any? %>">
```

Comment:
Ooh, typo

```suggestion
<div class="form-group <%= "form-group-error" if form.errors[:city].any? %>">
```

Reply:
Wow!! Great spot!! Wondering why the test suite didn't bring this up 🤔 will check the test coverage next